browser/extensions/brave_extension_provider.cc

```diff
@@ -96,7 +96,9 @@ bool BraveExtensionProvider::IsVetted(const Extension* extension) {
     // Test ID: Brave HTTPS Everywhere Updater
     "bhlmpjhncoojbkemjkeppfahkglffilp",
     // Test ID: Brave Tor Client Updater
-    "ngicbhhaldfdgmjhilmnleppfpmkgbbk"
+    "ngicbhhaldfdgmjhilmnleppfpmkgbbk",
+    // Test ID: Brave Sync Extension
+    brave_sync_extension_id,
```

Comment:
I think this should go at the top of the list with the other brave extension ids